unit-tests/test-phase.js

```diff
@@ -224,4 +223,19 @@ describe('phase', () => {
       assert.isFunction(phase.triggerGC);
     });
   });
+
+  describe('timeoutError', () => {
+    it('error thrown', () => {
+      let errorFunc = () => {
+        phase.timeoutError();
+      };
+      assert.throw(errorFunc, 'Test timeout exceeded ' + OPTIONS.timeout + 'ms');
```

Comment:
Whitespace after `let`. Still wondering if we can do `assert.throw(phase.timeoutError, ...)`.